google/cloud/pubsub/internal/tracing_message_batch.cc

```diff
@@ -91,12 +92,13 @@ auto MakeChild(
 }
 
 Spans MakeBatchSinkSpans(Spans message_spans) {
-  int64_t max_otel_links = 128;
+  size_t max_otel_links =
+      internal::CurrentOptions().get<pubsub::MaxOtelLinkCountOption>();
```

Comment:
Blegh... is there a way to get the options explicitly?  Or are they completely lost in the call stack by this point?

Reply:
Yeah that would be better. Fixed. Refactored so the class takes an options.